### DISCUSSION TITLE
Docs

### DISCUSSION THREAD

**Discussion by dhooper6430** [akvorado/akvorado#9] [Q&A]
Hi,

Would it please be possible to write a document on how to add a new router as a flow-exporter into akvorado once the router is exporting net-flow data?  I've looked into the docs you have available but I've not been able to setup my own routers and the package successfully.  Unfortunately I'm not very familiar with go, docker, Kafka etc, but I really like the UI interface you have created.

Kind Regards,

Daniel Hooper

**Comment by vincentbernat** [comment]
There is nothing to do on Akvorado side. Ask your router to send flows to Akvorado and it should work. If not, look at the troubleshooting section of the documentation.
